types/trim/index.d.ts

```diff
@@ -3,7 +3,7 @@
 // Definitions by: Steve Jenkins <https://github.com/skysteve>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare function Trim(str: string): string;
+declare function Trim(str: string | { trim(): string }): string;
```

Comment:
Is this supposed to be `trim` or `toString`? From the PR title I expected `toString`.
(I looked at the test you linked and it seems to show that an object can have a `trim` property equal to `null`?)